src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,78 @@
 package core.basesyntax;
 
 public class SalaryInfo {
+    static final int INDEX_START_DAY_DATE = 0;
+    static final int INDEX_END_DAY_DATE = 2;
+    static final int INDEX_START_MONTH_DATE = 3;
+    static final int INDEX_END_MONTH_DATE = 5;
+    static final int INDEX_START_YEAR_DATE = 6;
+    static final int INDEX_END_YEAR_DATE = 10;
```

Comment:
Create formatter for parsing the date. [Click here for more info](https://docs.oracle.com/javase/tutorial/datetime/iso/format.html). Also, it should be constant!